Libplanet/Net/Messages/Neighbours.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Libplanet.Net.Protocols;
+using NetMQ;
+
+namespace Libplanet.Net.Messages
+{
+    internal class Neighbours : Message
+    {
+        public Neighbours(IEnumerable<Peer> found)
+        {
+            if (found.Count() > int.MaxValue)
+            {
+                throw new ArgumentOutOfRangeException(
+                    nameof(found),
+                    $"The number of found peers can't exceed {int.MaxValue}.");
+            }
+
+            Found = found.ToList();
+        }
+
+        public Neighbours(NetMQFrame[] body)
+        {
+            int foundCount = body[0].ConvertToInt32();
+            Found = body.Skip(1).Take(foundCount)
+                .Select(f => DeserializePeer(f.ToByteArray()))
+                .ToList();
+        }
+
+        public List<Peer> Found { get; }
+
+        protected override MessageType Type => MessageType.Neighbours;
+
+        protected override IEnumerable<NetMQFrame> DataFrames
+        {
+            get
+            {
+                yield return new NetMQFrame(
+                    NetworkOrderBitsConverter.GetBytes(Found.Count));
+
+                foreach (Peer peer in Found)
+                {
+                    yield return new NetMQFrame(SerializePeer(peer));
```

Comment:
If the maximum value of found peer is `int.MaxValue`, can we send it as a message?
Shouldn't we divide it into chunks?

Reply:
`Found` is built in `RoutingTable.Neighbours()`, which returns maximum 32 peers.

Reply:
So shouldn't we check it's not over 32 [here](https://github.com/planetarium/libplanet/pull/353/files/6fbcae192e88d7092205041933d5df946e55c936#diff-bdc8d19aacbb2ffb302c9361c70c5b1fR14), too?

Reply:
Actually 32 is a value that might be changed due to protocol setting, so the line you mentioned is just used to prevent mistakes of users.